third_party/pcsc-lite/naclport/server/src/google_smart_card_pcsc_lite_server/global.h

```diff
@@ -38,6 +43,30 @@ namespace google_smart_card {
 // already been initialized.
 void InitializeAndRunPcscLiteServer();
 
+class PcscLiteServerGlobal final {
+ public:
+  explicit PcscLiteServerGlobal(pp::Instance* pp_instance);
+  ~PcscLiteServerGlobal();
+  void Detach();
+
+  static const PcscLiteServerGlobal* GetInstance();
+
+  void InitializeAndRunPcscLiteServer();
```

Comment:
"PcscLiteServer" part duplicates the part of the class name. Maybe rename it to "StartDaemonThread"?